backoffice/backoffice/workflows/tests/cassettes/TestAuthorWorkflowViewSet.test_validate_not_valid_record.yaml

```diff
@@ -1,4 +1,28 @@
 interactions:
+- request:
```

Comment:
es responses should not be recorded. in the `conftest.py` could you rename in the `ignore_hosts` `opensearch` to  `es` or alternatively find all instances of `es` and rename the to `opensearch`, and then rerun the tests? (my bad on this one, forgot to change this naming during the monorepoing)